### ISSUE TITLE
403 on sending test result [WRU] [/api/v1/quicktest/results]

### ISSUE THREAD

**Issue by r0bs** [corona-warn-app/cwa-quicktest-onboarding#83]
## Describe the bug

On sending results, I started to receive 403 responses about an hour ago. There seems to be some kind of authorisation issue with my account, that suddenly occurred on the server side.

Before everything worked perfectly, returning 204 responses. 

My certificate name: covidbs-wru.schnelltestportal.de-Server-01c37d1f8a33f63c00855fbe0e8b4f63

## Expected behaviour

Get a 204 on a valid request.

**Comment by jetwes** [comment]
Same here. Massive Problems since 13:00 
I have over 500 Tests in Backlog already.

**Comment by brkoglu** [comment]
Same here since 13:00 approx.

**Comment by jetwes** [comment]
On production. Desaster. All clients and customers are going crazy..

**Comment by Ein-Tim** [comment]
cc @dsarkar, @thomasaugsten

Incident?

**Comment by thomasaugsten** [comment]
I thinks @ascheibal can help here

**Comment by Ein-Tim** [comment]
@thomasaugsten It seems like there is a current incident. Can you open an internal investigation?

**Comment by jetwes** [comment]
Is there any ETA on this topic. Is there a status page or something like that?

**Comment by Ein-Tim** [comment]
@jetwes 

There is no status page. I hope @thomasaugsten started an internal investigation and this will be fixed soon.

**Comment by vaubaehn** [comment]
I hope, @ascheibal reads here. In most cases we don't get any reply from him.

**Comment by jetwes** [comment]
I have confirmation from my contact at T-Systems. The issue is known and is under investigation

**Comment by ascheibal** [comment]
I forwarded the issue to the operations team.
@vaubaehn: thanks for blaming me. API Partners should have the necessary contacts to address urgent production issues. As you mentioned, I'm probably the wrong person to address those.

**Comment by vaubaehn** [comment]
> I forwarded the issue to the operations team. @vaubaehn: thanks for blaming me. API Partners should have the necessary contacts to address urgent production issues. As you mentioned, I'm probably the wrong person to address those.

@ascheibal 
I'm sorry. But from my experiences in the past, it looked like that serious/important issues got rather ignored. Maybe we can change this in the future?

**Comment by novoelgy** [comment]
Dear All, I am sorry for the trouble, ths isuse should be fixed, please check again.

**Comment by jetwes** [comment]
Looking good! Thank you! Now i have to take care on my backlog. 1500 quicktests are waiting...

**Comment by Ein-Tim** [comment]
@brkoglu & @r0bs 

Can you confirm the fix?

**Comment by brkoglu** [comment]
Looks also good for me, thx!

**Comment by r0bs** [comment]
@brkoglu: works on WRU now

thx! :)

**Comment by vaubaehn** [comment]
@novoelgy 
Thanks for fixing on behalf of the user community.
As one could think that server maintenance caused the issue, is there any chance that these works on PROD servers can be done during a night shift in the future?
See also https://github.com/corona-warn-app/cwa-app-android/issues/968#issuecomment-986925177 for another sample incident.

**Comment by dsarkar** [comment]
Internal Tracking ID: [EXPOSUREAPP-11006](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11006)

**Comment by AmazingTurtle** [comment]
I have those issues every now and then. I got access to WRU yesterday. According to docs there should be a certificate whitelisting daily at 12:00 and 16:30, however I had to wait until ~14:00 and then my 403s went away. I was able to post test results to the quicktest server. But today when getting back to work and opening my project unchanged, trying to post test results again (no change of code at all) I get 403s again. There's not much to troubleshoot from my side. I checked everything multiple times and it should be fine. There's no hints as to what I could possibly be doing wrong. Is it the API again that causes issues?
